src/signalr/azext_signalr/key.py

```diff
@@ -0,0 +1,19 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from azext_signalr.signalr.models import KeyType
+
+
+def signalr_key_list(client, resource_group_name, signalr_name):
+    return client.list_keys(resource_group_name, signalr_name)
+
+
+def signalr_key_renew(client, resource_group_name, signalr_name, key_type):
+    if key_type == 'primary':
+        key_type = KeyType.primary
+    else:
+        key_type = KeyType.secondary
+    return client.regenerate_key(resource_group_name, signalr_name, key_type, polling=False)
```

Comment:
Change this to:
```
sdk_no_wait(True, client.regenerate_key, resource_group_name, signalr_name, key_type)
```
https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/util.py#L251

Reply:
If I use sdk_no_wait, the output will be raw. Thus, when I run `key renew`, it will raise error with `TypeError: <urllib3.response.HTTPResponse object at 0x0539B310> is not JSON serializable`